package.json

```diff
@@ -70,5 +71,12 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "*.js": [
+      "npm run lint",
+      "npm run prettier",
+      "npm run ci:test -- --passWithNoTests"
+    ]
```

Comment:
lint-staged is managed via `.lintstagedrc.js`, and is already running during pre-commit:

https://github.com/smeijer/testing-playground/blob/07eb868be89e2819c8040b19cdaa854fd73fb783/.lintstagedrc.js#L5-L4 

I rather not have the tests running on a pre-commit hook, as that can slow down development drastically. Github actions take care of testing when opening the pull-request. I think for testing that is good enough.

Reply:
I think it can be beneficial to run tests before committing.

If you look at `kcd-scripts`, the `pre-commit`-script is doing that too
https://github.com/kentcdodds/kcd-scripts/blob/master/src/config/lintstagedrc.js

Reply:
I didn't see the file. For now we can add and if in future we see that is slow we can thing something different

Reply:
Ok, let's add `test` (not `ci:test`) to `.lintstagedrc.js` than :)

`ci` scripts are only meant for github actions. `test` is what you should run prior to commit/push.